edurange_refactored/templates/dashboard/student_scenario.html

```diff
@@ -67,7 +67,7 @@ <h5 id="progress_s"> Points Earned: {{ progress['score'] }} / {{ progress['total
                 <div class="container mt-3">
                     <div class="card-body">
                         <div class="container">
-                            <iframe src="{{ guide }}" title="Codelab" style="width:1100px; height:1000px;"></iframe>
+                            <iframe src="{{ "data:text/html;charset=utf-8,%s" % guide }}" title="Guide" style="width:1100px; height:1000px;"></iframe>
```

Comment:
Do we need to be putting the guide in an iframe? Or could we have more formatting flexibility if it was inline html? Not sure. This works great but could be prettier.